### PR TITLE
33152- Added migrations to drop `vewCompletedCheckCount` and `vewIncompleteCheckCount`

### PR DESCRIPTION
**Pull request by danwilliams**
They both relied on now removed `check.data` but weren't being used in the admin app anyway

### CODE REVIEW

**Comment by jon-shipley** [comment]
If these views are not being used in the admin app perhaps we should drop them instead?

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis they are used by reporting users regularly, so will not be dropped.

**Comment by GuyHarwood** [comment]
views are used by Power BI users

**Comment by GuyHarwood** [comment]
@danwilliams feedback from the service manager indicates that these 2 views are used frequently and currently throw an error due to the inclusion of the `check.data` column.

Please could you modify this PR to `ALTER` the views and remove usage of that column.  Then we can push it through to master 😎

**Comment by GeorgeChatzigiannis** [comment]
@danwilliams seems like `vewPupilActivity` and `vewNotStartedCheckCount` need to be tackled as part of this PR?

File: admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ALTER VIEW [mtc_admin].[vewCompletedCheckCount] AS
+  SELECT c.pupil_id, COUNT(c.id) AS [CompletedCheckCount]
+  FROM [mtc_admin].[check] c
+    LEFT JOIN [mtc_admin].checkResult ck ON (c.id = ck.check_id)
```
Comment (jon-shipley) [inline admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql]: NB: I am surprised that `ck` is short for the `c`heck `r`esult table.  Why not use `cr`?

File: admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ALTER VIEW [mtc_admin].[vewCompletedCheckCount] AS
+  SELECT c.pupil_id, COUNT(c.id) AS [CompletedCheckCount]
```
Comment (jon-shipley) [inline admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql]: This appears to be the number of pupils with completed checks.  Perhaps rename the alias for clarity?

**Comment by GuyHarwood** [comment]
the rollback migrations are failing because the `data` column was removed a while ago.
i think in this particular situation you will have to omit the undo, as its to fix an already broken view

**Comment by GuyHarwood** [comment]
closing temporarily to lower build agent demand

**Pull request merged**